app/src/main/java/com/gh4a/utils/RxUtils.java

```diff
@@ -175,8 +178,17 @@ public void onDismissed(Snackbar snackbar, int event) {
                                 }
                             }
                         })
-                        .setAction(R.string.retry, view -> retryProcessor.onNext(0))
-                        .show();
+                        .setTextMaxLines(3)
+                        .setAction(R.string.retry, view -> retryProcessor.onNext(0));
```

Comment:
I'd prefer this to happen in an else branch below; it's a little confusing right now with action being set here and overwritten below.